NatureArea.py

```diff
@@ -55,6 +107,8 @@ def set_descriptions(self):
 
         National park format: "national park in Sweden".
         Reserve format: "nature reserve in <län>, Sweden".
+
+        :return: nothing
```

Comment:
In general if they don't return anything you can probably leave the return bit of the docstring out. Nothing that needs fixing though